changelog/std-experimental-checkedint.dd

```diff
@@ -0,0 +1,22 @@
+New: $(REF Checked, std, experimental, checkedint), a lightweight and highly
```

Comment:
Don't put links in the subject/title line. The whole line is turned into a link to the body, and you can't have links within links. You can put the REF in the body.